src/components/StoriesList/Pagination.tsx

```diff
@@ -0,0 +1,68 @@
+"use client";
+
+import { usePathname, useSearchParams } from "next/navigation";
+
+interface Props {
+  total_pages: number;
+}
+
+export default function Pagination({ total_pages }: Props) {
+  const params = Object.fromEntries(useSearchParams().entries());
+  if (!params.page) {
+    params.page = "1";
+  }
+  const page_number = parseInt(params.page);
+  const pages = Array.from({ length: total_pages }, (_, i) => i + 1);
+
+  return (
+    <>
+      <ul className="inline-flex justify-center gap-4 align-middle text-lg text-sciquelMuted">
+        <li>Page</li>
+        {page_number !== 1 && (
+          <li>
+            <a
+              className=""
+              href={`${usePathname()}?${new URLSearchParams({
+                ...params,
+                page: String(page_number - 1),
+              }).toString()}`}
+            >
+              {"<"} Previous
+            </a>
+          </li>
+        )}
+        {pages.map((page) => (
```

Comment:
I think this works fine for small number of pages, but if the number of pages is let's say 30, this might get out of hand pretty quickly. I suggest testing this with the API call set to a small page size to simulate having a large number of pages. Then, you can omit the middle  pages with ellipses (...).

![image](https://github.com/SciQuel/SciQuel/assets/17174688/760d5c40-c339-45b9-b60c-986c9c52b08a)

![image](https://github.com/SciQuel/SciQuel/assets/17174688/7885d5c7-a6d9-436a-972b-88075eed82c3)